### ISSUE TITLE
INPUT LABEL references must be valid [Accessibility issue]

### ISSUE THREAD

**Issue by RajshreejadhavIBM** [IBM/carbon-components-angular#1472]
Login and go to https://ecamstage14.multicloud-ibm.com/lite/insights/ridashboard
Use DAP to scan

Description: 

The id dropdown-0 referenced by the 'for' attribute of a LABEL element is not a valid FORM INPUT element or an element with id dropdown-0 is either null or does not exist.

![dropdown name](https://user-images.githubusercontent.com/70168592/91083134-66dcfc80-e667-11ea-8fb9-fa63cb7fe611.png)
<img width="594" alt="Screenshot 2020-08-25 at 12 11 04 AM" src="https://user-images.githubusercontent.com/70168592/91083185-7c522680-e667-11ea-9c2f-432b60e65bd5.png">

**Comment by Donisius** [comment]
Which DAP ruleset are you using and what version of carbon-components-angular are you using? It doesn't seem to be an issue in the latest version 3 and version 4

**Comment by cal-smith** [comment]
@RajshreejadhavIBM I can't access your link. Could you provide the info the @Donisius requested?

**Comment by RajshreejadhavIBM** [comment]
version details in package.json 

<img width="434" alt="Screenshot 2020-09-01 at 10 29 15 AM" src="https://user-images.githubusercontent.com/70168592/91796532-10128c80-ec3e-11ea-9001-eb68d9822da5.png">

**Comment by Donisius** [comment]
Is it still an issue in the latest version 3?

**Comment by RajshreejadhavIBM** [comment]
@Donisius yes.

**Comment by Donisius** [comment]
@RajshreejadhavIBM are you sure? I believe the `for` from the dropdown inputs have been removed some time ago which was what was causing the issue. I attached a screenshot of the storybook example for version `3.54.0`, the `for` in the input is no longer there and the DAP issues are storybook related.
![Screen Shot 2020-09-09 at 1 42 17 PM](https://user-images.githubusercontent.com/45505172/92634441-52714480-f2a2-11ea-828d-391b4713d68c.png)
